pandas/tests/io/test_sql.py

```diff
@@ -98,6 +99,7 @@
                     `TextCol` TEXT,
                     `DateCol` DATETIME,
                     `IntDateCol` INTEGER,
+                    "IntDateOnlyCol" INTEGER,
```

Comment:
you are using a different quoting here, that's probably the reason for the failure

Reply:
Thanks! I'll fix it.